### ISSUE TITLE
Incorrect error appears if `github` module function promise rejects

### ISSUE THREAD

**Issue by AlexanderMoskovkin** [DevExpress/testcafe#731]
Create a test that uses the [github](https://www.npmjs.com/package/github) module.
Wrap any function from the module that fails with a `Promise` and run it.
The report contains a strange error:
`The action element is expected to be an <iframe>.`

TestCafe version is `0.2.3-alpha`

*Example*
```js
import Github from 'github';

function createRef () {
    var github = new Github({
        version:  '3.0.0',
        protocol: 'https',
        host:     'api.github.com',
        timeout:  5000
    });

    github.authenticate({
        type:  'oauth',
        token: 'sometoken'
    });

    return new Promise((resolve, reject) => {
        github.gitdata.createReference({}, function (err, res) {
            if(err)
                reject(err);
            else
                resolve(res);
        });
    });
}

fixture `myFixture`;

test('myTest', async t => {
    await createRef();
});
```

*Report*
```
$ testcafe chrome D:/Temp/test1/test.js
 Running tests in:
 - Chrome 52.0.2743 / Windows 7 0.0.0

 myFixture
 × myTest

   1) The action element is expected to be an <iframe>.

      Browser: Chrome 52.0.2743 / Windows 7 0.0.0



 1/1 failed (0s)

```

The `createReference` function calling fails with the following error:
```
{ [Error: Empty value for parameter 'user': undefined]
  defaultMessage: 'Bad Request',
  message: 'Empty value for parameter \'user\': undefined',
  code: '400',
  status: 'Bad Request',
  headers: undefined }

```

**Comment by inikulin** [comment]
This one looks like a serious blocker, any news on it?

**Comment by helen-dikareva** [comment]
I'm working on this. I think it's not really serious, don't worry

**Comment by helen-dikareva** [comment]
This behavior is caused throwing Error in module `error-stack-parser`
https://github.com/stacktracejs/error-stack-parser/blob/04b29272bff2047496b822d165f7c0b0fd211b31/dist/error-stack-parser.js#L74
The module `callsite-record` doesn't handle this scenario, but I think it should return empty or null object in this case.
Steps to reproduce in `testcafe` (error which can't be parsed by `error-stack-parser`):
test.test.js
```
import error from './error';

function func () {
    return new Promise((resolve, reject) => {
        reject(new error.BadRequest());

    });
}

fixture `myFixture`;

test('myTest', async t => {
    await func();
});
```
error.js (simplified version of https://github.com/mikedeboer/node-github/blob/8af72a317c91ca114f194f3a535202b1d2e4cf62/lib/index.js#L169)
```
var Util = require('util');

exports.HttpError = function () {
    Error.call(this, 'Bar request error');
    this.message = 'Bar request error';
    this.code    = 400;
};
Util.inherits(exports.HttpError, Error);

var error = (function () {
    return function () {
        exports.HttpError.call(this);
    };
})();
Util.inherits(error, exports.HttpError);

exports.BadRequest = error;
```

**Comment by helen-dikareva** [comment]
Part with `callsite-record` fixed in `3.2.1` version
